cardano-testnet/src/Testnet/Defaults.hs

```diff
@@ -107,6 +110,76 @@ defaultAlonzoGenesis = do
         Nothing -> Left $ AlonzoGenErrTooMuchPrecision r
         Just s -> return s
 
+    maxTxExUnits = Api.toAlonzoExUnits
+                     $ Api.ExecutionUnits
+                         { Api.executionSteps = 100000000000
+                         , Api.executionMemory = 1400000000
+                         }
+    maxBlockExUnits = Api.toAlonzoExUnits
+                        $ Api.ExecutionUnits
+                            { Api.executionSteps = 20000000000
+                            , Api.executionMemory = 62000000
+                            }
+    apiCostModels =
+      let pv1 = Api.AnyPlutusScriptVersion Api.PlutusScriptV1
+          pv2 = Api.AnyPlutusScriptVersion Api.PlutusScriptV2
+          pv3 = Api.AnyPlutusScriptVersion Api.PlutusScriptV3
+      in mconcat [ Map.singleton pv1 defaultV1CostModel
+                 , Map.singleton pv2 defaultV2CostModel
+                 , Map.singleton pv3 defaultV3CostModel
+                 ]
+    defaultV1CostModel = Api.CostModel
```

Comment:
are those V1 and V2 values the same as in default genesis in cardano-api? Maybe it would be better to reference values from the default genesis there instead of redefining them here?

Same with V3 - maybe it would be better to just update single elements in the list instead?